app/AppManifest.json

```diff
@@ -1,8 +1,9 @@
 [
     {
-        "Name": "seatadjuster",
+        "Name": "myvehicleapp",
```

Comment:
Rename myvehicleapp to sampleapp in tha AppManifest.json 
- Also the MyVehicleApp(VehicleApp) to SampleApp(VehicleApp)